packages/retail-ui/components/DropdownContainer/DropdownContainer.tsx

```diff
@@ -57,12 +57,14 @@ export default class DropdownContainer extends React.Component<DropdownContainer
   }
 
   public componentWillMount() {
-    const htmlPosition = getComputedStyle(document.documentElement).position;
-    const bodyPosition = getComputedStyle(document.body).position;
+    const { body, documentElement: docEl } = document;
+    const htmlPosition = getComputedStyle(docEl).position;
+    const bodyPosition = getComputedStyle(body).position;
 
-    if (htmlPosition !== 'static' || bodyPosition !== 'static') {
-      this.setState({ hasStaticRoot: false });
-    }
+    const hasLimitedHeightRoot = docEl.offsetHeight !== docEl.scrollHeight;
```

Comment:
```suggestion
    const hasLimitedHeightRoot = docEl.clientHeight !== docEl.scrollHeight;
```
Тут вроде лучше подойдет `clientHeight`, т.к. он не включает бордеры.

Reply:
`documentElement.clientHeight` - высота видимой области окна, и не меняется при изменении высоты у `html`/`body`. Только при непосредственном изменении размера окна браузера и наличием скроллбара.

Уверен, что `margin`/`padding` для `html` уже давно никто не задаёт, и ими можно пренебречь.